CHANGES.rst

```diff
@@ -309,6 +309,8 @@ astropy.io.votable
 astropy.modeling
 ^^^^^^^^^^^^^^^^
 
+- Proper unit support for tabular models. [#6529]
```

Comment:
Perhaps `add unit support ...` as I don't think there was any before.

Reply:
Oh, in that case, should this be under "new feature" in v3 then?

Reply:
Oh, I see what you mean - it's currently under "Bug fixes". Yes, I think it should go under "New features".